tracing/src/main/java/com/palantir/remoting3/tracing/AsyncSlf4jSpanObserver.java

```diff
@@ -195,7 +196,7 @@ public static AsyncSlf4jSpanObserver of(
     @Override
     public void doConsume(Span span) {
         if (logger.isTraceEnabled()) {
-            logger.trace("{}", ZipkinCompatSpan.fromSpan(span, endpoint).toJson());
+            logger.trace("{}", UnsafeArg.of("span", ZipkinCompatSpan.fromSpan(span, endpoint).toJson()));
```

Comment:
delete the `toJson` bit

Reply:
Tried this but the tests then fail because they didn't use an objectmapper to serialize the args..